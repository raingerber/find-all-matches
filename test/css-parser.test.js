/* eslint-env jest */

const {getOpeningTagName} = require('../__test__/index')

describe('getOpeningTagName', () => {
  it('should return the first html tagName from the string', () => {
    expect(getOpeningTagName('<div><span></span></div>')).toBe('div')
  })
  it('should throw if the string does not contain an html tag', () => {
    expect(() => getOpeningTagName('<!-- <div></div> -->')).toThrowErrorMatchingSnapshot()
  })
})
