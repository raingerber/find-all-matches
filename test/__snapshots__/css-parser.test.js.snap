// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getOpeningTagName should throw if the string does not contain an html tag 1`] = `"Input HTML does not contain a valid tag"`;
